MapboxDirections/MBVoiceInstruction.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+@objc(MBVoiceInstruction)
+public class VoiceInstruction: NSObject, NSSecureCoding {
+    
+    public let distanceAlongGeometry: CLLocationDirection
```

Comment:
`CLLocationDistance` would be a more suitable type for this property than `CLLocationDirection`.